master/docs/manual/cfg-wwwhooks.rst

```diff
@@ -150,7 +150,7 @@ The parameters are:
                 ...))
 
 :guilabel:`Which events would you like to trigger this webhook?`
-    Leave the default -- ``Just the push [tag]  events`` -- other kind of events are not currently supported.
+    Leave the default -- ``Just the push [tag], create, release,  events`` -- other kind of events are not currently supported.
 
```

Comment:
Please write a paragraph explaining that push and pull_requests events are generating normal change events with list of file changed, while create and release events generate "dummy" changes, which does not reflect code changes, but rather useful event to start a build upon